joomla/joomla-cms
# Convert all remaining templates to HTML5

Pull Request for Improvement.
#### Summary of Changes

Basic conversion of all site and admin templates to HTML5 (some where still XHTML 1.0 Transitional).
#### Testing Instructions
1. Install patch
2. Check if pages in all templates (normal, component popup - ex: mail/print, error pages, offline and login) site and admin are working properly.

Check also the code changes.
#### Observations

All should now start with this code:

``` html
<!DOCTYPE html>
<html lang="en-gb" dir="ltr">
<head>
[...]
    <meta charset="utf-8" />
[...]
```

Suggestions and/or improvements are welcomed.
